docs/wxm-devices/D1-bundle/D1-troubleshooting.mdx

```diff
@@ -76,7 +78,10 @@ Please make sure you have followed [Best Practices for Proper Station Installati
 
 If you are still experiencing issues, please contact us through [support](https://support.weatherxm.com).
 
-### Station is reporting "Battery is Low".
+### Device Alert "Station Battery Level is Low"
```

Comment:
Add a `:` please ----> `Device Alert: "Station Battery Level is Low"`

Also, move this above "Temperature looks higher than expected" so that all Device Alerts are at the start of this section together.